nameko/rpc.py

```diff
@@ -350,10 +351,16 @@ def __call__(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply.result()
 
-    def async(self, *args, **kwargs):
+    def call_async(self, *args, **kwargs):
         reply = self._call(*args, **kwargs)
         return reply
 
+    def async(self, *args, **kwargs):  # pragma: no cover
```

Comment:
Hmm, I already tried that (`py.test --amqp-uri amqp://192.168.99.100:5672`), seems like it's hardcoded somewhere (or default value is used). I've ended up just `sed`-ing every `localhost` to my `docker-machine` ip, will try to figure out what's going on when I'll get a chance